### PR TITLE
feat: HS-187: Added Pix Bank Transfer

### PR DESCRIPTION
**Pull request by ArushKapoorJuspay**
**Details -**

```
payment_method - bank_transfer
payment_method_type - pix
```

**Confirm Body -**

```
"payment_method": "bank_transfer",
"payment_method_type": "pix",
"payment_method_data": {
  "bank_transfer": {
    "pix": {}
  }
}
```

### CODE REVIEW

File: src/Utilities/PaymentUtils.res [hunk 1]
```diff
@@ -295,3 +295,13 @@ let updateDynamicFields = (arr: Js.Array2.t<PaymentMethodsRecord.paymentMethodsF
...
+
+let getPaymentMethodName = (~paymentMethodType, ~paymentMethodName) => {
+  if paymentMethodType == "bank_debit" {
+    paymentMethodName->Js.String2.replace("_debit", "")
+  } else if paymentMethodType == "bank_transfer" {
+    paymentMethodName->Js.String2.replace("_transfer", "")
+  } else {
+    paymentMethodName
+  }
+}
```
Comment (prafulkoppalkar) [inline src/Utilities/PaymentUtils.res:307]: can you check how we were doing this for the existing bank_transfer methods ?
we are not reverting this back for those right .. please make use of the same here too

Comment (ArushKapoorJuspay) [reply src/Utilities/PaymentUtils.res:307]: For the existing bank transfers, it is handled in different files, ACHBankDebit and so on, PaymentMethodWrapper is not used

Comment (prafulkoppalkar) [reply src/Utilities/PaymentUtils.res:307]: okay since you are already on this.. can you make those files also utilise this function

Comment (prafulkoppalkar) [reply src/Utilities/PaymentUtils.res:307]: bumping this up

File: src/Components/DynamicFields.res [hunk 2]
```diff
@@ -19,7 +19,10 @@ let make = (
     PaymentMethodsRecord.getPaymentMethodTypeFromList(
       ~list,
       ~paymentMethod,
-      ~paymentMethodType,
+      ~paymentMethodType=PaymentUtils.getPaymentMethodName(
```
Comment (arun-mi) [inline src/Components/DynamicFields.res:22]: This method as I saw was removing the "transfer" part correct? Doing a lookup after removing doesn't seem optimal right? Also this isnt used for display is it?

**Review by arun-mi — APPROVED** [review]

**Pull request closed without merging**